tests/casestmt/tincompletecaseobject2.nim

```diff
@@ -1,12 +1,5 @@
 discard """
 cmd: "nim check $file"
-errormsg: "not all cases are covered; missing: {A, B}"
```

Comment:
`errmsg` currently overrides `nimout`, so I got rid of both in this test and inlined the messages.